File: Corredor/Digitales/Go/instrumento.go
package main

import "fmt"

type Instrumento struct {
	nombre string
	tipo   string
	costo  float64
}

func NuevoInstrumento(nombre, tipo string, costo float64) Instrumento {
	return Instrumento{nombre, tipo, costo}
}

func (i Instrumento) Mostrar() {
	fmt.Printf("Instrumento: %s | Tipo: %s | Costo: $%.2f\n", i.nombre, i.tipo, i.costo)
}

File: Corredor/Digitales/C#/C#/Instrumento.cs
namespace EscuelaMusica
{
    public class Instrumento
    {
        public string Nombre { get; set; }
        public string Tipo { get; set; }
        public double Costo { get; set; }

        public Instrumento(string nombre, string tipo, double costo)
        {
            Nombre = nombre;
            Tipo = tipo;
            Costo = costo;
        }

        public void Mostrar()
        {
            System.Console.WriteLine($"Instrumento: {Nombre} | Tipo: {Tipo} | Costo: ${Costo}");
        }
    }
}